internal/controller/websocket.go

```diff
@@ -0,0 +1,63 @@
+package controller
+
+import (
+	"Tradeasy/internal/services/stock_exchange"
+	"encoding/json"
+	"github.com/gin-gonic/gin"
+	"github.com/gorilla/websocket"
+	"log"
+	"time"
+)
+
+const TimeInterval = time.Duration(5)*time.Millisecond*1000
```

Comment:
use time.seconds directly.

Reply:
Name variable as for what purpose is this being used.

Reply:
renamed to stockTimeInterval and added time.second